modules/hosted-che/partials/proc_providing-a-devfile.adoc

```diff
@@ -5,7 +5,7 @@
 [id="providing-a-devfile_{context}"]
 = Providing a devfile
 
-Providing a devfile in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
+Providing a link:https://devfile.io/[devfile] in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
 This ensures that the workspace contains everything users need to effectively review the pull request, such as plug-ins, development commands, and other environment setup.
 
 The link:https://github.com/eclipse/che-docs/blob/master/devfile.yaml[Che documentation repository devfile] is an example of a well-defined and effective devfile to start workspaces from.
```

Comment:
The link: https://github.com/eclipse/che-docs/blob/master/devfile.yaml[Che documentation repository devfile] is an example of a well-defined and effective devfile to start workspaces.

Reply:
ignore the "link:" part. Please reword the remainder of the sentence as suggested.